WooCommerce/Classes/ViewModels/Order Details/OrderDetailsViewModel.swift

```diff
@@ -491,13 +491,13 @@ extension OrderDetailsViewModel {
                     viewModel: CustomFieldsListViewModel(customFields: customFields),
                     onBackButtonTapped: {
                     // Restore the hidden navigation bar
-                    viewController.navigationController?.setNavigationBarHidden(false, animated: true)
+                    viewController.navigationController?.setNavigationBarHidden(false, animated: false)
                 })
             )
 
             // Hide the navigation bar as `CustomFieldsListView` will create its own toolbar.
-            viewController.navigationController?.setNavigationBarHidden(true, animated: true)
-            viewController.navigationController?.pushViewController(customFieldsView, animated: true)
+            viewController.navigationController?.setNavigationBarHidden(true, animated: false)
+            viewController.navigationController?.pushViewController(customFieldsView, animated: false)
```

Comment:
IMO we can keep the animation for the push transition - my suggestion was to remove only the the transition of the navigation bar visibility.

Reply:
I tested in iPhone and iPad cases and I agree that keeping the push looks better, thanks. https://github.com/woocommerce/woocommerce-ios/pull/14068/commits/d513b5400bffcd894a11166547d417702fc34aad